[2025-10-23T04:50:23.672+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-23T04:50:23.738+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Fraud_detection_training.cleanup_resources manual__2025-10-23T04:50:15.691869+00:00 [queued]>
[2025-10-23T04:50:23.763+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Fraud_detection_training.cleanup_resources manual__2025-10-23T04:50:15.691869+00:00 [queued]>
[2025-10-23T04:50:23.764+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-10-23T04:50:23.795+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): cleanup_resources> on 2025-10-23 04:50:15.691869+00:00
[2025-10-23T04:50:23.806+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=100) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-23T04:50:23.810+0000] {standard_task_runner.py:72} INFO - Started process 101 to run task
[2025-10-23T04:50:23.813+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Fraud_detection_training', 'cleanup_resources', 'manual__2025-10-23T04:50:15.691869+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmpobtj4i1a']
[2025-10-23T04:50:23.816+0000] {standard_task_runner.py:105} INFO - Job 9: Subtask cleanup_resources
[2025-10-23T04:50:23.922+0000] {task_command.py:467} INFO - Running <TaskInstance: Fraud_detection_training.cleanup_resources manual__2025-10-23T04:50:15.691869+00:00 [running]> on host 661dc816a375
[2025-10-23T04:50:24.114+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Rogelio Franco' AIRFLOW_CTX_DAG_ID='Fraud_detection_training' AIRFLOW_CTX_TASK_ID='cleanup_resources' AIRFLOW_CTX_EXECUTION_DATE='2025-10-23T04:50:15.691869+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-23T04:50:15.691869+00:00'
[2025-10-23T04:50:24.115+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-10-23T04:50:24.164+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-10-23T04:50:24.166+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'rm -f /app/tmp/*.pkl']
[2025-10-23T04:50:24.188+0000] {subprocess.py:99} INFO - Output:
[2025-10-23T04:50:24.200+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-10-23T04:50:24.264+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-10-23T04:50:24.265+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=Fraud_detection_training, task_id=cleanup_resources, run_id=manual__2025-10-23T04:50:15.691869+00:00, execution_date=20251023T045015, start_date=20251023T045023, end_date=20251023T045024
[2025-10-23T04:50:24.351+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-23T04:50:24.402+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-23T04:50:24.407+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
